### ISSUE TITLE
Make the portal theme Screen Reader compatible

### ISSUE THREAD

**Issue by gggodhwani** [cbgaindia/ckanext-openbudgetsin_theme#37]
Currently, some of the pages aren't compatible with Screen Reader mostly because of missing alternative text for images and labels for search bars. We need to edit HTML templates to add appropriate help text to fix most of these issues.

Few WAVE(http://wave.webaim.org) failing links:
http://wave.webaim.org/report#/https://openbudgetsindia.org/
http://wave.webaim.org/report#/https://openbudgetsindia.org/dataset?res_format=PDF
http://wave.webaim.org/report#/https://openbudgetsindia.org/organization/
http://wave.webaim.org/report#/https://openbudgetsindia.org/budget-basics/

Source: https://groups.google.com/forum/#!topic/datameet/AOmYmWZ4LqE

**Comment by vjayakumar** [comment]
@gggodhwani - not to be a pain, but are there any plans of getting this data in csv / spreadsheet format as well? Because the default Government table with its complex, merged, tiered headings will break the screen reader!

**Comment by shansub15** [comment]
Yes Vaishnavi, rightly pointed out. The merged headers will play a spoilt sport.

Rakesh gives a simple lesson in his blog for the beginners to understand the format that would support csv export also.
http://www.maxability.co.in/2014/07/data-tables-accessibility/

In case of complex tables

It would be more a tbody, thead, tfoot, id, for headers within the table structure. For more information please go through the below link. This would help the screen reader to recognise the headers, read them out and then into data.

http://usability.com.au/2005/06/accessible-data-tables-2005/#complex

I need to try out if such data tables can be exported into a csv.

**Comment by imakshayverma** [comment]
@vjayakumar @shansub15 have pushed some changes fixing the major errors which were highlighted in the WAVE accessibility tool. I understand it is an on going process. But we probably have fixed the first set of errors. #41 
It would be great if you could review them, and let us know if there are any major issues left.

**Comment by shansub15** [comment]
Thanks Akshay

**Comment by vjayakumar** [comment]
Doing a check and will get back with feedback. 

In the meantime, how about an **[accessibility statement](https://www.nomensa.com/blog/2009/writing-an-accessibility-statement)** somewhere on website with a feedback link to [flag accessibility issues](https://data.gov.in/accessibility-statement)? Apart from showcasing intent, (and softening irate user rants!) that would be the best realtime usability check.

Secondly, if some content is only available in image form, is there anyway one could search for only accessible content? Scanned PDFs could always be flagged in some way - like pending OCR - so that a screenreader doesn't waste time?
